NguyenVanViet2004/FE-GlowUp
# Implement light/dark mode switching feature

## Related issues
Resolved https://github.com/NguyenVanViet2004/FE-GlowUp/issues/33

## Summary
- Create and add `colorSlice` to redux store
- Implement light/dark mode switching feature
- Refactor UI elements to implement light/dark mode switching feature 

## How to test
- Navigate to `Hồ sơ cá nhân` tab 
- Toggle icon `Moon/Sun` to switching between  light/dark mode
- Verify all UI elements display correctly when click switching  between  light/dark mode
- Reload app to check theme

## Concern points

## Screenshots
| Dark mode | Light mode |
| --- | --- |
|![Simulator Screenshot - iPhone 16 Plus - 2024-12-03 at 12 33 14](https://github.com/user-attachments/assets/18b02c2f-cfdc-4c26-b92c-5b8ace9f2aac)|![Simulator Screenshot - iPhone 16 Plus - 2024-12-03 at 12 33 16](https://github.com/user-attachments/assets/fe4402d1-17a6-4d26-bdbe-8dcce9e9ac19)|
